### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ZakkFast**
created navbar

### CODE REVIEW

File: template/style.css [hunk 1]
```diff
@@ -1,131 +1,465 @@
 :root {
-    /* --color1: #05f2af; */
-    --primary: #01ABAA;
-    --secondary: #FEA680;
-    --alert: #FEDC3D;
-    --light: #efefee;
-    --dark: #333;
-    --dark2: #000;
+  /* --color1: #05f2af; */
```
Comment (rashir01) [inline template/style.css]: commented code

File: template/style.css [hunk 2]
```diff
@@ -1,131 +1,465 @@
...
 body {
-    padding: 0;
-    margin: 0;
-    background-color: var(--light);
+  padding: 0;
+  margin: 0;
+  background-color: var(--light);
+}
+:root {
+  /* --color1: #05f2af; */
```
Comment (rashir01) [inline template/style.css]: commented code

File: template/style.css [hunk 3]
```diff
@@ -1,131 +1,465 @@
...
+.bg-alert {
+  background-color: var(--alert);
+}
+.bg-light {
+  background-color: var(--light);
+}
+.bg-dark {
+  background-color: var(--dark);
+}
+.bg-dark2 {
```
Comment (rashir01) [inline template/style.css]: can we come up with a better name? bg-dark2 doesnt really tell me what it is going to be used for

File: template/style.css [hunk 4]
```diff
@@ -1,131 +1,465 @@
...
+  color: var(--alert);
+}
+.text-light {
+  color: var(--light);
+}
+.text-dark {
+  color: var(--dark);
+}
+.text-dark2 {
+  color: var(--dark2);
```
Comment (rashir01) [inline template/style.css]: same as naming comment above

File: template/style.css (hunk 1 above)
Comment (ZakkFast) [reply template/style.css]: Removed

**Review by rashir01 — APPROVED** [review]

**Pull request merged**